pipelines/orthofinder_modules.nf

```diff
@@ -51,14 +60,18 @@ process orthoFinderFactory {
 */
 process runOrthoFinder {
 
-    label 'slurm_default'
+    label 'slurm_64Gb_32C'
+
+    scratch true
+
+    publishDir "${params.ref_destination_cloud}/${x}", mode: 'copy', overwrite: 'true'
 
     input:
-    val x
+    path x
 
     shell:
     """
-    ${params.orthofinder_exe} -t 32 -a 8 -f ${params.ref_destination}/${x}
+    ${params.orthofinder_exe} -t 32 -a 8 -f ${x}
```

Comment:
Do the `t` and `a` parameters could be not hard-coded here?

Reply:
They don't have to be, but for a robust production pipeline, it would be easiest - the idea being that the workflow and resources are optimal for the production environment. This could be seen as an improvement with the idea that the process resource could be dynamic too - in a similar way: https://github.com/Ensembl/ensembl-compara/pull/455#discussion_r898015589